ecs-configuration/logstash/conf.d/logstash-502-filter-zeek-network-ssl.conf

```
@@ -13,8 +13,8 @@ filter {
           "[tls][client_issuer]" => "[tls][client][issuer]"
           "[tls][client_subject]" => "[tls][client][subject]"
           "[tls][issuer]" => "[tls][server][issuer]"
-          "[tls][ja3]" => "[tls][client][ja3]"
-          "[tls][ja3s]" => "[tls][server][ja3s]"
+          "[tls][ja3]" => "[tls][client][ja3][hash]"
```

Per above mapping template suggestion, revert this to.
```suggestion
          "[tls][ja3]" => "[tls][client][ja3]"
```